### DISCUSSION TITLE
Next ImageMagic 7.1.0-46 release date

### DISCUSSION THREAD

**Discussion by oliviermetienne** [ImageMagick/ImageMagick#5450] [Development] [answered]
Hello,
A bug was recently fix under https://github.com/ImageMagick/ImageMagick/issues/5422
As part of the resolution of this call, I was told that the 7.1.0.46 would be release last weekend. I have just check but it still has not.
Can you confirm the release date?
The reason is that I am writing a book where ImageMagick is used and I would like the readers to download the most recent version which fixes the bug.

You help is greatly appreciated.

**Comment by dlemstra** [comment]
That release is probably going to happen soon. If you are writing a book that needs to be printed I am pretty sure the new release will be published before your book is printed.

**Reply by oliviermetienne** [reply]
Thank you very much indeed, very much appreciated,  In this case, I will go ahead as planned.

**Comment by urban-warrior** [comment, marked as answer]
ImageMagick 7.1.0-46 was released earlier today.

**Reply by oliviermetienne** [reply]
Very well indeed, that will give me plenty of time to double and test my instruction all of next week.
Much appreciated.
